src/assets/images/check.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="10" viewBox="0 0 14 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Thank you for doing SVG rather than an image

Reply:
svgs are da way